tracer/test/test-applications/integrations/Samples.GrpcLegacy/ClientWorker.cs

```diff
@@ -63,7 +63,7 @@ public async Task ExecuteAsync(CancellationToken stoppingToken)
             var client = new Greeter.GreeterClient(callInvoker);
 
             await SendVerySlowRequestAsync(client);
-            delay = Task.Delay(6_000); // longer than the slow request duration
+            delay = Task.Delay(1_000); // longer than the slow request duration
```

Comment:
I worry that these numbers are getting a bit tight for CI, but we'll see how it goes

Reply:
Yeah that is my fear as well